pallets/loans/src/benchmarking.rs

```diff
@@ -158,6 +159,20 @@ where
 	let pool_id: PoolIdOf<T> = Default::default();
 	let pool_account = pool_account::<T>(pool_id.into());
 	let pal_pool_id: T::PoolId = pool_id.into();
+
+	T::AssetRegistry::register_asset(
+		Some(CurrencyId::AUSD.into()),
+		orml_asset_registry::AssetMetadata {
+			decimals: 18,
+			name: "MOCK AUSD TOKEN".as_bytes().to_vec(),
+			symbol: "MckAUSD".as_bytes().to_vec(),
+			existential_deposit: 0.into(),
+			location: None,
+			additional: CustomMetadata::default(),
+		},
+	)
+	.expect("Registering pool currency must work");
+
```

Comment:
ℹ️  We now need to register AUSD before we create a test pool, since the latter extrinsic checks for the presence of the pool currency metadata in the asset registry